cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -29,6 +29,7 @@ r.Streaming.FullyLoadUsedTextures=1
 [/Script/EngineSettings.GameMapsSettings]
 GameDefaultMap=/Game/Scenes/default/Maps/DefaultMap.DefaultMap
 EditorStartupMap=/Game/Scenes/default/Maps/DefaultMap.DefaultMap
+GlobalDefaultGameMode=/Script/SpearSim.SpearSimGameMode
```

Comment:
Nice. I wonder how difficult it is to make `ASpectatorPawn` receive input and update the viewport while the game is paused. If this is easy to do, then we would effectively a nice debug camera that could move freely even when our simulation is paused and we're waiting for Python input.

This functionality already works in Debug builds by activating the debug camera through the debug console, but the default debug camera is too sensitive to keyboard input so it is hard to navigate, and it would be nice to enable this functionality on Shipping builds.